index.js

```diff
@@ -49,6 +49,9 @@ const draggableEvents = {
 }
 const deviceEvents = isTouchDevice ? draggableEvents.touch : draggableEvents.desktop
 
+const pixelRatio = typeof window !== 'undefined' && typeof window.devicePixelRatio !== 'undefined'
+  ? window.devicePixelRatio : 1
+
```

Comment:
this could be summarized as:
`const pixelRatio = typeof window !== 'undefined' && window.devicePixelRatio || 1;`